bin/MadGraph5_aMCatNLO/cards/production/pre2017/14TeV/DMScalar_top_tWChan_Mchi1_Mphi100/DMScalar_top_tWChan_Mchi1_Mphi100_proc_card.dat

```diff
@@ -0,0 +1,34 @@
+#************************************************************
+#*                     MadGraph5_aMC@NLO                    *
+#*                                                          *
+#*                *                       *                 *
+#*                  *        * *        *                   *
+#*                    * * * * 5 * * * *                     *
+#*                  *        * *        *                   *
+#*                *                       *                 *
+#*                                                          *
+#*                                                          *
+#*         VERSION 2.2.2                 2014-11-06         *
+#*                                                          *
+#*    The MadGraph5_aMC@NLO Development Team - Find us at   *
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#************************************************************
+#*                                                          *
+#*               Command File for MadGraph5_aMC@NLO         *
+#*                                                          *
+#*     run as ./bin/mg5_aMC  filename                       *
+#*                                                          *
+#************************************************************
+import model DMScalar
+define p = g u c d s u~ c~ d~ s~
```

Comment:
it looks to me that the proton definition is 4F but PDF (263000) and maxjetflavor (5) are for 5F, can you please clarify what the model needs and adjust accordingly?

Reply:
For our process, the t-channel is produced in 4F while the tW-channel in 5F accordingly with the SM single top, i.e. the b quark is included in the proton definition as in:

https://github.com/dpinna/genproductions/blob/155961e099f5a147b040f3daa8925c978851dc7e/bin/MadGraph5_aMCatNLO/cards/production/pre2017/14TeV/DMScalar_top_tWChan_Mchi1_Mphi100/DMScalar_top_tWChan_Mchi1_Mphi100_proc_card.dat#L30

An example of single top t-channel 5F I have found (I cannot find the previous examples used for the various channels) is:

https://github.com/cms-sw/genproductions/blob/mg26x/bin/MadGraph5_aMCatNLO/cards/production/13TeV/st_tch_5f_ckm_NLO/st_tch_5f_ckm_NLO_proc_card.dat

I hope this answer the questions and motivates the definitions used in our cards.

Reply:
I think there is an inconsistency for the tW case, because the proton definition doesn't include the b quark, making if effectively a 4F definition, but the PDF and the maxjetflavor are for 5F

Reply:
The proton definition does include the b as written in the cards as specified in line 24 and 30 (the one linked before):

define p = g u c d s u~ c~ d~ s~
...
define p = p b  b~

Reply:
ok, then can you please define it on a single line please?

Reply:
sure, done and the pull requested has been updated